src/test/resources/org/javarosa/core/model/instance/geojson/feature-collection-extra-toplevel.geojson

```diff
@@ -0,0 +1,21 @@
+{
+    "type": "FeatureCollection",
+    "features": [
+        {
+            "type": "Feature",
+            "geometry": {
+                "type": "Point",
+                "coordinates": [
+                    102,
+                    0.5
+                ]
+            },
+            "foo": "bar",
```

Comment:
Good to make this not exactly the same as the property in `properties` that we do want. It took me a good 5 minutes to understand what was happening here!

Reply:
Good call. It almost looks like I wandered off before finishing this last commit, sorry. Renamed the property to `ignored` and force pushed.